pyproject.toml

```
@@ -20,3 +20,40 @@ addopts = """
 fail_under = 0
 show_missing = true
 exclude_lines = ["pragma: no cover", "raise NotImplementedError"]
+
+[tool.poetry]
+name = "qc-data-preparation"
+version = "0.18.1"
+description = ""
+authors = ["lc3267 <melvin.strobl@kit.edu>"]
```

There are some more authors involved:

```suggestion
authors = [
    {name="Eileen Kuehn", email="eileen.kuehn@kit.edu"},
    {name="Melvin Strobl", email="melvin.strobl@kit.edu"},
    {name="Christof Wendenius", email="christof.wendenius@kit.edu"},
]
```